### PR TITLE
Practice create new file

### PR DESCRIPTION
**Pull request by baveeshna**
Trying to commit a new file
@jineshmd 
- [ ] read
- [ ] comment

### CODE REVIEW

File: bad-weather.md [hunk 1]
```diff
@@ -0,0 +1 @@
+[forecast](http://www.accuweather.com/en/in/bangalore/204108/weather-forecast/204108) :umbrella:
```
Comment (mattas1) [inline bad-weather.md:1]: did u attend the upgrade meeting ?

**Comment by jineshmd** [comment]
Looks good! Fix upgrade meeting isssue.

File: bad-weather.md (hunk 1 above)
Comment (baveeshna) [inline bad-weather.md:1]: No power.

**Comment by jineshmd** [comment]
Good answer. Merging

**Pull request merged**